.github/pull_request_template.md

```diff
@@ -1,28 +1,47 @@
-## Brief Information
+<!-- 
+🚨NOTE🚨 Please complete the following tasks before submitting a pull request:
 
-This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):
+- [ ] Read the CONTRIBUTING doc. see: https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md
+- [ ] Sign the CLA. see: https://cla-assistant.io/hypertrons/hypertrons-crx
+-->
 
-- [ ] build
-- [ ] ci
-- [ ] docs
-- [ ] feat
-- [ ] fix
-- [ ] perf
-- [ ] refactor
-- [ ] test
+### Brief
 
-Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):
+This pull request is in the type of:
+<!-- 
+- `build`: Changes related to the build system, such as dependencies, packaging, and compiling.
+- `ci`: Changes related to continuous integration, such as configuration files and scripts.
+- `docs`: Changes related to documentation, such as README files and API documentation.
+- `feat`: New features or enhancements to existing features.
+- `fix`: Bug fixes or patches to existing features.
+- `perf`: Changes related to performance improvements.
+- `refactor`: Changes that improe code quality or structure without changing functionality.
+- `test`: Changes related to testing, such as adding or modifying unit tests. 
 
-- keyword #xxx
+Ref: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type
 
-## Details
-<!-- What did you do in this PR?  -->
+Example:
+
+    This pull request is in the type of: build
+-->
+
+Related issues:
+<!-- 
+Link the issue to the PR.
+
+Ref: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
```

Comment:
In my veiw, ref links are better in the form of `[description](url)` and to be placed in the content rather in the comment. In that way, GitHub newcomers who cannot edit the PR description to see the markdown source are able to visit the references easily (just click the link on the text), so can learn these tips by reading existing PRs  before actually creating their first PR in this repo.

Reply:
Newcomers still need to check the preview to get the link. And I believe most newcomers don't how to edit the PR text after posting or checking the preview carefully.

It also seems verbose if it includes much irrelevant text in the output post. But it's OK if you can take that.